cli/pcluster/config/validators.py

```diff
@@ -136,6 +136,9 @@ def fsx_validator(section_key, section_label, pcluster_config):
     if fsx_export_path and not fsx_import_path:
         errors.append("When specifying 'export_path', the 'import_path' option must be specified")
 
+    if not fsx_section.get_param_value("storage_capacity") and not fsx_section.get_param_value("fsx_fs_id"):
+        errors.append("When specifying 'fsx' section, the 'storage_capacity' option must be specified")
```

Comment:
Should we also check for the specific storage capacity as outlined by FSx: `For Lustre file systems, valid values are 1,200, 2,400, 3,600, then continuing in increments of 3600 GiB.`

We should at least provide this info in the error message so the user does not have to look it up again

Reply:
That's done here https://github.com/aws/aws-parallelcluster/blob/develop/cli/pcluster/config/validators.py#L194